LearnositySDK/Request/Init.cs

```diff
@@ -412,21 +412,13 @@ private void setServiceOptions()
 
                         JsonObject questionsApiActivity = new JsonObject();
 
-                        List<string> signatureList = new List<string>();
-
-                        signatureList.Add(this.securityPacket.getString("consumer_key"));
-                        signatureList.Add(domain);
-                        signatureList.Add(this.securityPacket.getString("timestamp"));
-
                         if (Tools.array_key_exists("expires", this.securityPacket))
                         {
-                            signatureList.Add(this.securityPacket.getString("expires"));
                             questionsApiActivity.set("expires", this.securityPacket.getString("expires"));
                             questionsApi.remove("expires");
                         }
 
-                        signatureList.Add(this.securityPacket.getString("user_id"));
-                        signatureList.Add(this.secret);
+                      
```

Comment:
Will remove extra lines in my next commit

Reply:
https://github.com/Learnosity/learnosity-sdk-asp.net/pull/43/commits/8fdb49c3b0ae72b1ec615dd9ed1795a98f3a04f0